styles.css

```diff
@@ -46,16 +49,25 @@ nav > div{
 	border: 2px solid #777674;
 }
 
-.navSmallScreen{
-	display: none;
+
+.navigation > a{
+	text-decoration: none;
+}
+
+.navigation > a:visited{
+	color: #000000;
+}
+
+.navigation > a:link{
+	color: #000000;
```

Comment:
This didn't change anything in the inspector when I turned it off (or when I clicked the links)

Reply:
Strange it doesn't come up on my inspector.  But turning off manually in the code definitely has an effect in my browser.